static/examples.json

```diff
@@ -38031,26 +34596,19 @@
             "wedding essentials"
           ],
           "product_description": "escape in the luxurious lather of our ultra-creamy, iconic philosophy bath & shower gel, upgraded with our skin-hugging moisture complex for all-day hydration.",
-          "price": "36",
-          "discounted_price": "30.60",
+          "price": 36.0,
+          "discounted_price": 30.6,
           "size": [
             "16 oz",
             "32 oz"
           ],
           "color": [],
           "images": [
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_SG_Range_16oz_32oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_SG_hero_16oz_1500x1500_6a4372d4-3a7c-49ec-b4db-dab71390eac8.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_Ingredient_Multi_Infographic_SG_16oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHB_SG_24_Digital_Visual_Key_Claims_Model_1x1.psd-JPG-300dpi-1550px_743a1a6a-a80a-481a-9ae6-2f335f588fd3.webp?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_SG_macro_1500x1500_c0e1fd3b-b827-41a9-8ba7-5d8537d7a3e3.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_SG_Range_16oz_32oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
-            "//philosophy.com/cdn/shop/files/PHF_AG_24_Digital_SG_hero_16oz_1500x1500_6a4372d4-3a7c-49ec-b4db-dab71390eac8.jpg?width=1200",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
-            "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+            "https://philosophy.com/cdn/shop/files/PHF_AG_24_Digital_CPS_SG_Range_16oz_32oz_1x1.psd-JPG-300dpi-1550px.jpg?width=1200",
```

Comment:
are these the only images though? Might cause problems with the agent if we're not actually retrieving it all

Reply:
yes checked for all of them and it actually correctly retrieves all the images, but it also retrieves duplicates for some of them as well as the base64 placeholder